libs/aws/langchain_aws/function_calling.py

```diff
@@ -203,12 +214,23 @@ def _pydantic_parse(self, tool_call: ToolCall) -> BaseModel:
         ]
         return cls_(**tool_call["args"])
 
+def is_cache_point(cache_point: Any) -> bool:
+    return (
+        isinstance(cache_point, dict) 
+        and len(cache_point) == 1
+        and "cachePoint" in cache_point 
+        and isinstance(cache_point["cachePoint"], dict)
+        and len(cache_point["cachePoint"]) == 1
+        and "type" in cache_point["cachePoint"]
+    )
```

Comment:
Can simplify this check. Also, can move this to bedrock_converse.py file, since this will not be used outside.

```suggestion
def is_cache_point(cache_point: Any) -> bool:
    return (
        isinstance(cache_point, dict) 
        and "cachePoint" in cache_point
        and cache_point.get("cachePoint").get("type") is not None
    )
```